### ISSUE TITLE
Facilities: Table lacks scope

### ISSUE THREAD

**Issue by NiquiCara** [rubyforgood/abalone#538]
This table needs a scope definition. Since the headings are in the top row, it should probably be `scope="col"` to each of the `th` tags. 

The action buttons don't have a header at all. They need a header.

![table-scope](https://user-images.githubusercontent.com/45127691/102561589-191b5280-40a3-11eb-8240-820ddd1d0d59.PNG)

**Comment by metamoni** [comment]
Opened a PR for this
